### DISCUSSION TITLE
How expensive is a getBlock and setBlock call?

### DISCUSSION THREAD

**Discussion by Flopgop** [Minestom/Minestom#1707] [Help]
just curious

**Comment by KrystilizeNevaDies** [comment]
You can find this out via running the inbuilt benchmarks.

**Comment by emortaldev** [comment]
Can't be as bad as spigot!
ofc you also have [Block Batches](https://wiki.minestom.net/world/batch) to help with it if it's a decent amount of blocks

If you end up doing benchmarks it would be nice to know at what block count it becomes worth it to use block batches
